usagov/usagov-2021
# USAGOV-1367 - PR review, adding CSS spacing

https://cm-jira.usa.gov/browse/USAGOV-1367

## Description
As requested on the ticket, have this additional PR for adding spacing.
Requested: Can you please add 20px below the message block that says "There was a problem getting results for this address. Please check to be sure you entered a valid U.S. address."

## Type of Changes


## Testing Instructions
After merge go to either:
* http://localhost/elected-officials-results?input-city=GSAScanner&input-street=GSASc 
* or:
* https://cms-dev.usa.gov/elected-officials-results?input-city=GSAScanner&input-street=GSASc 
* confirm there is spacing under the requested text.

## Reviewer Reminders

- Reviewed code changes
- Reviewed functionality
- Security review complete or not required

## Post PR Approval Instructions

Follow these steps as soon as you merge the new changes.

1. Go to the [USAGov Circle CI project](https://app.circleci.com/pipelines/github/usagov/usagov-2021).
2. Find the commit of this pull request.
3. Build and deploy the changes.
4. Update the Jira ticket by changing the ticket status to `Review in Test` and add a comment. State whether the change is already visible on [cms-dev.usa.gov](http://cms-dev.usa.gov/) and [beta-dev.usa.gov](http://beta-dev.usa.gov/), or if the deployment is still in process.
